doc/Makefile

```diff
@@ -37,7 +39,12 @@ help:
 clean:
 	-rm -rf $(BUILDDIR)/*
 
-html:
+ext:
```

Comment:
Do you have a "netlify" option in Makefile? Can't find it

Reply:
Added.